modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/seedlist/HistoricalSnapshotOrdinalEntries.scala

```diff
@@ -0,0 +1,68 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.syntax.all._
+
+import org.tessellation.schema.SnapshotOrdinal
+
+import derevo.cats.eqv
+import derevo.derive
+import eu.timepit.refined.auto._
+import eu.timepit.refined.numeric.NonNegative
+import eu.timepit.refined.refineV
+import fs2.data.csv.{CellDecoder, DecoderError}
+
+@derive(eqv)
+case class HistoricalSnapshotOrdinalEntries(values: List[HistoricalSnapshotOrdinalEntry])
+
+object HistoricalSnapshotOrdinalEntries {
+
+  implicit val historicalSnapshotOrdinalEntriesCellDecoder: CellDecoder[Option[HistoricalSnapshotOrdinalEntries]] =
+    CellDecoder.stringDecoder.emap { s =>
+      val timeline = mkSnapshotOrdinalTimeline(s)
+
+      if (timeline.isDefined)
+        timeline.asRight
+      else
+        new DecoderError("Unable to parse the ordinal timeline").asLeft
+    }
+
+  private val valueDelimiter = "-"
+  private val entryDelimiter = ";"
+
+  def mkSnapshotOrdinalTimeline(value: String): Option[HistoricalSnapshotOrdinalEntries] = {
+    val entries = value.trim
+      .split(entryDelimiter)
+      .map(mkHistoricalSnapshotOrdinal)
+      .filter(_.isDefined)
+      .map(_.get)
+      .toList
+
+    if (entries.isEmpty)
+      none
+    else
+      HistoricalSnapshotOrdinalEntries(entries).some
+  }
+
+  private def mkHistoricalSnapshotOrdinal(value: String): Option[HistoricalSnapshotOrdinalEntry] = {
+    val splitValues = value
+      .split(valueDelimiter)
+      .map(_.trim.toLongOption)
+      .map {
+        _.flatMap(refineV[NonNegative](_).toOption)
+      }
+      .filter(_.isDefined)
+      .map(_.get)
+
+    if (value.contains("--") || splitValues.length =!= 2 || splitValues.head > splitValues.tail.head)
+      none
```

Comment:
would be great to notify somehow the user if there are errors in the seedlist, instead of just ignoring them